core/kazoo/src/kz_network_utils.erl

```diff
@@ -151,7 +152,8 @@ verify_cidr(IP, CIDR) when is_binary(CIDR) ->
     verify_cidr(IP, kz_term:to_list(CIDR));
 verify_cidr(IP, CIDR) ->
     Block = inet_cidr:parse(CIDR),
-    inet_cidr:contains(Block, inet:parse_address(IP)).
+    {'ok', IPTuple} = inet:parse_address(IP),
```

Comment:
I thought about it too.
It minimize changes, then the simpliest way is change [kz_network_utils:iptuple_to_binary/1](https://github.com/2600hz/kazoo/blob/master/core/kazoo/src/kz_network_utils.erl#L309) then it always return IPv4 address when it found 0:0:0:0:0:ffff:0:0/96 address.